images-list

```diff
@@ -1,14 +1,10 @@
-istio/sidecar_injector rancher/istio-sidecar_injector 1.5.9
```

Comment:
You can leave all the old ones because we will need them for more 1.5.x releases